server/portal/urls.py

```diff
@@ -114,7 +115,9 @@
     path('version/', portal_version),
 
     # health check
-    path('core/health-check', health_check)
+    path('core/health-check', health_check),
 
+    # internal docs
+    re_path(f'^{settings.INTERNAL_DOCS_URL}(?P<path>.*)$', serve_docs),
```

Comment:
One last note: if `INTERNAL_DOCS_URL` and `INTERNAL_DOCS_ROOT` is undefined, should the route be disabled?

Reply:
@rstijerina  - good point, fixed that in latest commit